src/main/java/hudson/scm/subversion/UpdateUpdater.java

```diff
@@ -167,7 +168,7 @@ private SVNInfo parseSvnInfo(File workspace) throws SVNException {
             } catch (SVNCancelException e) {
                 if (isAuthenticationFailedError(e)) {
                     e.printStackTrace(listener.error("Failed to check out " + location.remote));
-                    return null;
+                    throw (AbortException) new AbortException().initCause(e);
```

Comment:
`AbortException` intentionally does not have a constructor initializing `cause` becausing calling `initCause` on it is kind of pointless: the whole reason for using `AbortException` is to not print a stack trace.

And printing the stack trace of the `SVNCancelException` is probably too noisy, when this is a normal enough mistake.

So the whole `if` block body should probably just read

    throw new AbortException("Failed to check out " + location.remote);